CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added 
```

Comment:
Hi, first of all, thank you for your contribution. I do believe that this change makes sense but can you also make it for the version on `master`, so we keep the two evolving together?

Reply:
@salesfelipe Did not quite understand. Is it to update with the master?

Reply:
Master has a different version of this app, version 2.x, you are currently changing the version 3.x.  On this app, we evolve the 2 versions together, It would be nice if you apply the same changes that you made here to the version on master, in a different PR.

Reply:
Thanks  @salesfelipe . I will provide. Would that also include changes to the my-account app?

Reply:
No need for now... Thank you again 🙏